doc/source/myrocks/zenfs.rst

```diff
@@ -1,125 +1,241 @@
 
-     lsblk -o NAME,SIZE,ZONED
-     NAME        SIZE  ZONED
-     sda       247.9G  none
-     |-sda1    230.9G  none
-     |-sda2        1G  none
-     |-sda3       16G  none
-     sdb        15.5T  host-managed
+The zone storage model organizes the SSD into a set of zones that are uniform in size and uses the Zoned Namespaces (ZNS) technology. ZNS is optimized for an SSD and exposes this zoned block storage interface between the host and SSD. ZNS enables smart data placement. Writes are sequential within a zone.
 
+ZenFS is a file system plugin for RocksDB which uses the RocksDB file system to place files into zones on a raw zoned block device (ZBD). The plugin adds native support for ZNS, avoids garbage collection, and minimizes write amplification. File data is stored in a set of extents. Within a zone, extents are a contiguous part of the address space. Garbage collection is an option, but this selection can cause write amplification. 
 
-#. Create an auxiliary directory for the ZenFS, for example, ``/var/lib/mysql_aux``. The ZenFS auxiliary directory is a regular (POSIX) file system used internally to resolve file locks and shared access. There are no strict requirements for the auxiliary directory location but the directory must be write accessible for the `mysql:mysql` UNIX system user account. Each ZBD must have an individual auxiliary directory.
+ZenFS depends on the ``libzbd`` user library and requires a Linux kernel implementation that supports NVMe Zoned Namespaces. The kernel
+must be configured with `zone block device support
+enabled <https://zonedstorage.io/docs/linux/config>`__.
 
+Read the `Western Digital and Percona deliver
+Utrastar DC ZN540 Zoned Namespace SSD support for Percona Server for
+MySQL <https://documents.westerndigital.com/content/dam/doc-library/en_us/assets/public/western-digital/collateral/company/western-digital-zns-ssd-perconal-blogpost.pdf>`__ PDF for more information.
 
-#. Run ``/zenfs mkfs --zbd <short_block_device_name> --aux_path=<path_to_aux_zenfs_dir>`` in the context of the mysql:mysql account to initialize ZenFS on the ZBD.
 
-    .. sourcecode:: bash
+The following procedure installs Percona Server for MySQL and then configures 
+``--rocksdb-fs-uri=zenfs://dev:<short_block_device_name>`` for data storage.
 
-        zenfs mkfs --zbd=nvme1n2 --aux_path=/var/lib/mysql_aux
+.. note::
 
-    .. note::
+   The ``<block_device_name>`` can have a short name designation which is the ``<short_block_device_name>``. For the purposes of this document, the ``block_device_name`` is ``/dev/nvme0n2`` and the short name is ``nvme0n2``. 
+    
+For the moment, Percona provides a binary release for `Debian 11.1 <https://www.debian.org/releases/bullseye/debian-installer/index>`__. `Other Linux distributions <https://zonedstorage.io/docs/distributions/linux/>`__ are adding support for ZenFS, but Percona does not provide installation packages for those distributions yet.  
```

Comment:
Actually, we build binaries with support for ZenFS for 2 platforms:
1. Debian 11.1 out-out-of-the box (no changes required).
2. Ubuntu 20.04 Focal (users will have to manually install 5.11 HWE kernel patched with CAP_SYS_ADMIN fix)

On Focal, binaries with ZenFS support can still be run on standard 5.4 kernel provided that users do not enable ZenFS functionality (do not specify `--rocksdb-fs-uri` option)

Reply:
Updated.